### DISCUSSION TITLE
Welcome to CoreGPX Discussions!

### DISCUSSION THREAD

**Discussion by vincentneo** [vincentneo/GPXKit#87] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nwilsongme** [comment]
I am unable to archive with CorGPX (which is excellent and simple to use)

I am getting : CoreGPX framework not found

Any ideas ? 

IntermediateBuildFilesPath/UninstalledProducts/iphoneos/CoreGPX.framework" failed: No such file or directory

**Reply by nwilsongme** [reply]
problem solved

https://developer.apple.com/forums/thread/727525
